azurerm/internal/services/web/parse/slot_virtual_network_swift_connection.go

```diff
@@ -7,23 +7,23 @@ import (
 )
 
 type SlotVirtualNetworkSwitchConnectionId struct {
```

Comment:
Is it intentional that you did not rename the actual type's name as well? It seems to contain the same typo.

Reply:
I'll just do it :)

Reply:
Nope, missed it, good catch! 😄